src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeResponseMapper.java

```diff
@@ -38,6 +38,8 @@ public EnvelopeResponse toEnvelopeResponse(Envelope envelope) {
             envelope.getZipFileCreateddate(),
             envelope.getZipFileName(),
             envelope.getStatus(),
+            (envelope.getClassification() == null ? null : envelope.getClassification().toString()),
```

Comment:
I wish Java had `?.` operator like Kotlin ;)

Reply:
Envelope validation would needs to be done separately. Again when it is mandatory why we want to save null in db?

Reply:
You are right as we have a default value for classification now. This was from the initial implementation.